messaging/src/main/java/org/axonframework/queryhandling/GenericSubscriptionQueryUpdateMessage.java

```diff
@@ -63,6 +63,19 @@
         return new GenericSubscriptionQueryUpdateMessage<>((T) payload);
     }
 
+    /**
+     * Creates a {@link GenericSubscriptionQueryUpdateMessage} with the given {@code declaredType} and {@code exception}
+     * result.
+     *
+     * @param declaredType The declared type of the Subscription Query Update Message to be created
+     * @param exception    The exception describing the cause of an error
+     * @param <T>          type of the {@link GenericSubscriptionQueryUpdateMessage}
+     * @return a message containing exception result
+     */
+    public static <T> SubscriptionQueryUpdateMessage<T> asUpdateMessage(Class<T> declaredType, Throwable exception) {
```

Comment:
Just for my understanding here. This method is purely here to more easily test the `GrpcBackedQueryUpdateMessageTest`, right? 

But, next to that it can be used by users to not return a normal update through the `QueryUpdateEmitter`, but actually to emit an exceptional update. Something which currently can only be achieved if thy use the `GenericSubscriptionQueryUpdateMessage#asUpdateMessage(Object)` method wherein they provide a `ResultMessage` implementation which itself is exceptional. Correct?

So, it's not purely the fix you're doing here, but providing a simplified form to emit exceptional updates. If I follow correctly that is.

Reply:
This is correct. I've added this method for easier instantiation of exceptional query update messages. And it was pretty handy for test as well :P It goes in line with `<R> QueryResponseMessage#asResponseMessage(Class<R>, Throwable)`.